src/components/table-content-wrapper/table-content-wrapper.styles.js

```diff
@@ -7,6 +7,16 @@ export default ({ mixins, palette, typography }) => ({
     borderBottom: [2, 'solid', palette.color.grayDarker],
   },
   screenReadersOnly: mixins.screenReadersOnly,
+  captionMobileHidden: {
+    [mixins.maxMedia('md')]: {
+      ...mixins.screenReadersOnly,
```

Comment:
can we do `[mixins.maxMedia('md')]: mixins.screenReadersOnly` instead? Now we're only spreading one object

Reply:
Oh and btw nice a11y! 💪